EXPLAINER.md

```diff
@@ -502,7 +621,9 @@ Some other notes:
   finger is likely to be co-located with the current screen and window, not on
   the separate target screen.
 - ScreenInfo IDs generally follow patterns of other device information APIs.
+- New affordances for fullscreen requests on `Screens.onchange` events follows
```

Comment:
New affordances ... follows sounds a bit off. Does one of the following variants work?

New affordances ... follow
or
Having new affordances ... follows

Reply:
I changed it to "A new affordance ... follows the precedent"